src/main/java/frc/robot/subsystems/swerve/SwerveModule.java

```diff
@@ -26,24 +29,29 @@ public SwerveModule(int moduleNumber, int driveMotorID, int angleMotorID, int en
         this.encoderID = encoderID;
         this.angleOffSet = new Rotation2d(Math.toRadians(angleOffSetDegrees));
 
-        fields = new LogFieldsTable("Swerve Module " + this.moduleNumber);
-        io = new SwerveModuleIOFalcon(fields, this.driveMotorID, this.angleMotorID, this.encoderID);
+        fields = new FieldsTable("Swerve Module " + this.moduleNumber);
```

Comment:
I think it will be better to make the modules fields table a subtable of the Swerve fields table.